src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -18074,6 +18102,47 @@ instruct vabs2D(vecX dst, vecX src)
   ins_pipe(vunop_fp128);
 %}
 
+// --------------------------------- FABS DIFF --------------------------------
+
+instruct vabsd2F(vecD dst, vecD src1, vecD src2)
+%{
+  predicate(n->as_Vector()->length() == 2);
+  match(Set dst (AbsVF (SubVF src1 src2)));
```

Comment:
We now have aarch64_neon.ad, do you think we should put neon vector rules to that file, to keep aarch64.ad smaller?

Reply:
I've considered this.
But I feel a little bit unconsistent that only the new `fabd` is added into `aarch64_neon.ad`,
while other NEON intructions (i.e. `fabs`, `fsub`, `fdiv`, `fsqrt`, etc) are still in aarch64.ad.
And moving them all from aarch64.ad to aarch64_neon.ad deviates far from this patch.

I think I can put the close related `fabs` and `fabd` into aarch_neon.ad in this patch.
Is that OK?

Reply:
> But I feel a little bit unconsistent that only the new fabd is added into aarch64_neon.ad,
> while other NEON intructions (i.e. fabs, fsub, fdiv, fsqrt, etc) are still in aarch64.ad.
> And moving them all from aarch64.ad to aarch64_neon.ad deviates far from this patch.

Yes, I think when we introduced aarch64_neon.ad (m4), we just tried to keep that patch simple and would move other vector rules in future patches. Maybe Andrew can comment on this?